oak-auth-external/pom.xml

```diff
@@ -69,6 +69,16 @@
             <artifactId>org.osgi.service.component</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
which code changes requires this new compile classpath change?

Reply:
No code changes, these were listed as used undeclared dependencies by the dependency plugin.